scripts/build_snp.sh

```
@@ -0,0 +1,183 @@
+
+    yes "" | make olddefconfig
+    make -j$(nproc) deb-pkg
+    popd
+}
+
+build_ovmf() {
+    local build_dir=$1
+    pushd ${build_dir}
+
+    git clone --single-branch --depth 1 --branch edk2-stable202411 https://github.com/tianocore/edk2.git ovmf
+    cd ovmf
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)